### ISSUE TITLE
Object.fromEntries unusable

### ISSUE THREAD

**Issue by michaelmior** [facebook/flow#7183]
Using `Object.fromEntries([['a', 1]]);` produces the error below:

```
    3: Object.fromEntries([['a', 1]]);
              ^ Cannot call method `fromEntries` because property `fromEntries` is missing in statics of `Object` [1].
        References:
        [LIB] ..//static/v0.86.0/flowlib/core.js:38: declare class Object {
                                                                   ^ [1]
```

**Comment by wchargin** [comment]
`Object.fromEntries` is a nonstandard function in Firefox 63+ only (not
any other browsers and not Node). It has [a TC39 proposal][prop] but has
not yet been accepted.

If you wanted to submit a pull request to add support for this function
to Flow, you should modify `lib/core.js`:

https://github.com/facebook/flow/blob/0490dee17f323ab13cfcfd7b717a8db874775e49/lib/core.js#L38

But I’m not sure whether the maintainers would want to accept this PR.

[prop]: https://github.com/tc39/proposal-object-from-entries

**Comment by michaelmior** [comment]
Thanks @wchargin! I'll admit I didn't realize this was FF only. I could understand why this might not be accepted, so I'll just close this for now.

**Comment by bfreundOST** [comment]
Any news on if/when this might be added to node?

**Comment by wchargin** [comment]
@bfreundOST You’ll want to ask on the Node repo for that:
<https://github.com/nodejs/node/issues?utf8=%E2%9C%93&q=fromEntries>

**Comment by Hypnosphi** [comment]
Fixed with https://github.com/facebook/flow/pull/7435

**Comment by dnalborczyk** [comment]
this issue can be closed as it has been added: https://github.com/facebook/flow/blob/master/lib/core.js#L56

released in **v0.101.0**
